graylog-storage-elasticsearch7/src/main/java/org/graylog/storage/elasticsearch7/RemoteReindexingMigrationAdapterES7.java

```diff
@@ -32,4 +34,9 @@ public RemoteReindexMigration start(URI uri, String username, String password, L
     public RemoteReindexMigration status(String migrationID) {
         throw new UnsupportedOperationException("Remote reindexing migrations are not supported for elasticsearch");
     }
+
+    @Override
+    public IndexerConnectionCheckResult checkConnection(URI uri, String username, String password) {
+        throw new UnsupportedOperationException("Remote reindexing migrations are not supported for elasticsearch");
```

Comment:
I think we need some rewording to avoid misconceptions. How about:
"This operation should never be called. We remote-reindex into the DataNode that contains OpenSearch. This adapter only exists for API completeness"

Reply:
Good idea, done